### DISCUSSION TITLE
Support ticket response time?

### DISCUSSION THREAD

**Discussion by serranocirrus** [npm/feedback#741] [Website]
Hi, I sent in a support ticket (#1717778) a week ago but I haven't gotten a response, and I never saw an indicator for how long your normal response time is. Is this normal? Could somebody take a look at it please? thanks :) -anx

**Comment by MylesBorins** [comment]
I'm sharing with our support team to figure out what happened here.

Did you open this ticket from https://npmjs.com/support?

**Comment by serranocirrus** [comment]
Yup, got a response, thanks so much
